docs/user-manual/libaio.adoc

```diff
@@ -22,20 +22,39 @@ When using libaio, Apache ActiveMQ Artemis will always try loading these files a
 == Runtime dependencies
 
 If you just want to use the provided native binaries you need to install the required libaio dependency.
+You can install libaio using the following steps either as the root user or using `sudo`:
 
-You can install libaio using the following steps as the root user:
-
-Using yum, (e.g. on Fedora or Red Hat Enterprise Linux):
-
+* On Fedora, CentOS, Rocky Linux, Red Hat Enterprise Linux, etc.:
++
 ----
-yum install libaio
+dnf install libaio
 ----
 
-Using aptitude, (e.g. on Ubuntu or Debian system):
-
+* On Ubuntu, Debian, etc.:
++
 ----
-apt-get install libaio1
+apt install <package-name>
 ----
++
+The value for `<package-name>` will depend on which version of Linux you're using:
++
+
+|===
+|Linux Version |Package Name
+
+|Ubuntu 22.04 (and earlier)
+|libaio1
+
+|Ubuntu 24.04 (and later)
+|libaio1t64
```

Comment:
Does this still need the symlink workaround for them changing the .so name? If so should that be documented?